docs/resources/artifactory_virtual_conan_repository.md

```diff
@@ -0,0 +1,42 @@
+# Artifactory Virtual Conan Repository Resource
+
+Provides an Artifactory virtual repository resource, but with specific conan features. This should be preferred over the original
+one-size-fits-all `artifactory_virtual_repository`. 
+
+## Example Usage
+
+```hcl
+resource "artifactory_virtual_conan_repository" "foo-conan" {
+  key          = "foo-conan"
+  repo_layout_ref = "conan-default"
+  repositories = []
+  description = "A test virtual repo"
+  notes = "Internal description"
+  includes_pattern = "com/jfrog/**,cloud/jfrog/**"
+  excludes_pattern = "com/google/**"
+}
+```
+
+## Argument Reference
+
+Arguments have a one to one mapping with the [JFrog API](https://www.jfrog.com/confluence/display/RTF/Repository+Configuration+JSON). The following arguments are supported:
+
+* `key` - (Required)
+* `repositories` - (Required, but may be empty)
+* `description` - (Optional)
+* `notes` - (Optional)
+* `includes_pattern` - (Optional)
+* `excludes_pattern` - (Optional)
+* `repo_layout_ref` - (Optional)
+* `artifactory_requests_can_retrieve_remote_artifacts` - (Optional)
+* `key_pair` - (Optional) - Key pair to use for... well, I'm not sure. Maybe ssh auth to remote repo?
```

Comment:
It'd be great if someone could figure it out

Reply:
From UI, I see 'Key Pair' shown for maven repo type in advanced tab of virtual repo creation page. But, Not for conan, go, generic, etc.

Reply:
Yeah, there are definitely common attributes but also some very type specific attributes. Probably can't use one generic function for all virtual repos.

Reply:
Not for all of them, but for a lot of them - in fact, it's probably the exact same one as generic